pkgs/development/tools/google-java-format/default.nix

```diff
@@ -21,7 +21,12 @@ stdenv.mkDerivation rec {
     install -D ${src} $out/share/google-java-format/google-java-format.jar
```

Comment:
Shouldn't the jar keep the version in the file name?

Reply:
You're right, I thought it was expected to place it in some other directory, but it's simpler to just do `java -jar ${src}`